README.md

```diff
@@ -270,29 +270,32 @@ Whenever a change in the internal graph occurs.
 
 ## API
 ### `compiler = bankai(entry, [opts])`
-Create a new bankai instance. Takes either an entry file location, or an array
-of files.
+Create a new bankai instance from the route to JavaScript file. The following
```

Comment:
> Create a new bankai instance from the route to JavaScript file

I'm not sure how to parse this sentence 🙈

Reply:
lol, yeah fair

Let's redo it as:
> Create a new bankai instance. Takes a path to a JavaScript file as the first argument. (...)

Reply:
👌